docs/source/examples/checks/integrity/data_duplicates.ipynb

```diff
@@ -5,19 +5,28 @@
    "id": "06bdcdfc",
```

Comment:
<p>This is the opportunity to improve also the description of the data duplicates...</p><p>"Search for duplicate data in dataset."? (e.g. Checks for duplicate samples in the dataset)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/626/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>And also opporunity to improve this 8.81E-3% output</p>